### PR TITLE
Updated Airflow Docs

### PR DESCRIPTION
**Pull request by rohit-dimagi**
##### SUMMARY

* Updated Airflow Docs

##### ENVIRONMENTS AFFECTED

* ALL
##### ISSUE TYPE

- Change Pull Request

### CODE REVIEW

File: src/commcare_cloud/ansible/deploy_airflow.yml [hunk 1]
```diff
@@ -2,4 +2,6 @@
 - name: Airflow
   hosts: airflow
   roles:
-    - {role: airflow, tags: airflow}
+    - role: airflow
+      tags: airflow 
+      when: groups.airflow |length == 1
```
[the remark this answers is not in the document]
Comment (rohit-dimagi) [reply src/commcare_cloud/ansible/deploy_airflow.yml]: Added an assert message and a airflow_scheduler group.

**Comment by rohit-dimagi** [comment]
Update doc too.

File: src/commcare_cloud/ansible/deploy_airflow.yml [hunk 2]
```diff
@@ -1,5 +1,15 @@
 ---
+- hosts: airflow
```
Comment (snopoke) [inline src/commcare_cloud/ansible/deploy_airflow.yml]:
```suggestion
- hosts: airflow_scheduler
```

File: src/commcare_cloud/ansible/deploy_airflow.yml [hunk 3]
```diff
@@ -1,5 +1,15 @@
...
+  tasks:
+    - name: "Check host count in airflow group"
```
Comment (snopoke) [inline src/commcare_cloud/ansible/deploy_airflow.yml]:
```suggestion
    - name: "Check host count in airflow_scheduler group"
```

File: src/commcare_cloud/ansible/deploy_airflow.yml [hunk 4]
```diff
@@ -1,5 +1,15 @@
...
+      assert:
+        that: 
+          - groups.airflow |length == 1
```
Comment (snopoke) [inline src/commcare_cloud/ansible/deploy_airflow.yml]:
```suggestion
          - groups.airflow_scheduler |length == 1
```

File: src/commcare_cloud/ansible/deploy_airflow.yml [hunk 5]
```diff
@@ -1,5 +1,15 @@
...
+        fail_msg: "No of host in airflow group shouldn't exceed 1"
```
Comment (snopoke) [inline src/commcare_cloud/ansible/deploy_airflow.yml]:
```suggestion
        fail_msg: "Only 1 host is allowed in the airflow_scheduler group"
```

File: src/commcare_cloud/ansible/deploy_airflow.yml [hunk 6]
```diff
@@ -1,5 +1,15 @@
...
+
 - name: Airflow
   hosts: airflow
   roles:
-    - {role: airflow, tags: airflow}
+    - role: airflow
+      tags: airflow 
+      when: groups.airflow |length == 1
```
Comment (snopoke) [inline src/commcare_cloud/ansible/deploy_airflow.yml]: This change can be removed now

File: docs/howto/Airflow_setup.md [hunk 7]
```diff
@@ -0,0 +1,9 @@
+## Migrating Airflow
+1. Add the node information in inventory with `airflow` group assigned to it. (leave old host in airflow_scheduler group)
```
Comment (snopoke) [inline docs/howto/Airflow_setup.md]:
```suggestion
1. Add the node information in inventory with `airflow` group assigned to it. (leave old host in `airflow_scheduler` group)
```

**Comment by rohit-dimagi** [comment]
Updated Doc.

**Pull request merged**